app/containers/authorCvSection/index.tsx

```diff
@@ -0,0 +1,383 @@
+import * as React from "react";
+import { denormalize } from "normalizr";
+import { Dispatch, connect } from "react-redux";
+import { ConnectedAuthorShowState } from "../connectedAuthorShow/reducer";
+import { LayoutState } from "../../components/layouts/records";
+import { withStyles } from "../../helpers/withStylesHelper";
+import { AppState } from "../../reducers";
+import { CurrentUser } from "../../model/currentUser";
+import { authorSchema, Author } from "../../model/author/author";
+import { profileSchema, Profile, CVInfoType } from "../../model/profile";
+import Icon from "../../icons";
+import AwardForm, { AwardFormState } from "../../components/authorCV/awardForm";
+import { addAuthorCvInfo, removeAuthorCvInfo } from "../../actions/author";
+import EducationForm, { EducationFormState } from "../../components/authorCV/educationForm";
+import ExperienceForm, { ExperienceFormState } from "../../components/authorCV/experienceForm";
+import ExperienceItem from "../../components/authorCV/experienceItem";
+import EducationItem from "../../components/authorCV/educationItem";
+import AwardItem from "../../components/authorCV/awardItem";
+import { ActionCreators } from "../../actions/actionTypes";
+import alertToast from "../../helpers/makePlutoToastAction";
+const styles = require("./authorCvSection.scss");
+
+interface AuthorCvSectionState {
+  isOpenAwardForm: boolean;
+  isOpenEducationForm: boolean;
+  isOpenExperienceForm: boolean;
+  isLoadingAwardForm: boolean;
+  isLoadingEducationForm: boolean;
+  isLoadingExperienceForm: boolean;
+}
+
+interface AuthorCvSectionProps {
+  layout: LayoutState;
+  author: Author;
+  authorShow: ConnectedAuthorShowState;
+  currentUser: CurrentUser;
+  profile: Profile;
+  dispatch: Dispatch<any>;
+}
+
+export function getFormattingDate(year: string, month: string) {
+  return year.concat("-", month);
```

Comment:
Use year+"-"+month or \`${year}-${month}\` for performance.
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/String/concat